src/modules/filelist/HeaderCell.jsx

```diff
@@ -12,7 +12,8 @@ const HeaderCell = ({
   attr,
   order = null,
   defaultOrder,
-  onSort
+  onSort,
+  className
```

Comment:
nit: little tips, it will be much easier to review and to rebase if you don't add anything after the end of an object, like so : `defaultOrder, ClassName, onSort`